web/post-restoration-transition-application/app/components/Shares/AddEdit.vue

```diff
@@ -37,8 +42,12 @@ const resetData = () => {
       hasRightsOrRestrictions: false,
       maxNumberOfShares: undefined,
       parValue: undefined,
-      priority: shareClasses.value.length + 1,
-      series: []
+      priority: shareClasses.value.length + 1
+    }
+    if (props.isSeries && editingSeriesParent.value !== -1) {
```

Comment:
There's a few `props.isSeries && editingSeriesParent.value !== -1`. Will this component break if this happens / it's not an expected scenario? Or is that a valid scenario and this component handles it?

Reply:
It's not an expected scenario, if it is a series the editingSeriesParent must be set

Reply:
Can you add a comment for this up by the props?